plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1804,6 +1807,9 @@ public void setTableProperties(ConnectorSession session, ConnectorTableHandle ta
         try {
             transaction.commitTransaction();
```

Comment:
why not `commitTransaction(transaction);` here?
when one should be used and when the other?